JavaScript/keylogger.js

```diff
@@ -3,6 +3,8 @@ window.addEventListener('keydown', e => {
     pressedKeys.push(e.key)
 });
 
+
```

Comment:
Hello Mohamed, why you put two blank lines? can you remove this lines please?

Reply:
My bad, I will erase this two blank lines